client/src/components/Dashboard.js

```diff
@@ -211,7 +211,7 @@ class Dashboard extends Component {
                   due_date,
                   submission_date
                 } = document;
-
+                const labelDueDate = moment(due_date);
                 const dueDate = moment(due_date).format("DD/MM/YYYY");
```

Comment:
const labelDueDate should be called 'dueDate'
const dueDate should be called 'labelDueDate'

Reply:
dueDate is a string and should be in this format("DD/MM/YYYY") to be displayed in the dashboard, while the labelDueDate has been created only to be used to calculate the duration otherwise it will not be counted if it is a string. to be honest, this is the only way I have found to overcome this problem.